hospital_simulator/app/(DashboardLayout)/components/service-requests/create-service-request-dialog.tsx

```diff
@@ -174,7 +174,8 @@ function createServiceRequestBundle(firstName: string, lastName: string, conditi
     }
     const performerIdentifier = performer.identifier[0]
 
-    const patientBsn = Date.now();
+    let patientBsn = "" + Date.now()
+    patientBsn = patientBsn.substring(patientBsn.length - 9, patientBsn.length)
```

Comment:
This code doesn't create "valid" BSNs (as those must comply with the "elfproef"). If they only care about the length of the BSN, then the solution seems fine. 

Maybe check with Dirk if they need that?

Reply:
Maybe we should take the extra step to generate actually valid test BSNs? They are generally in the 9999 range (https://www.rvig.nl/sites/default/files/2023-05/Test%20burgerservicenummers%20en%20A-nummers%20inclusief%20omnummertabel.ods)?

Then they are never confused as being actual "real" BSNs

Reply:
Seems like the best option to me.

Reply:
Let's do that in a follow-up PR